src/main/java/perficient/academic/universityapplication/models/Student.java

```diff
@@ -0,0 +1,27 @@
+package perficient.academic.universityapplication.models;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.PrimaryKeyJoinColumn;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotNull;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import perficient.academic.universityapplication.enums.UserType;
+
+@Entity
+@NoArgsConstructor
+@PrimaryKeyJoinColumn(name = "id")
+public class Student extends User
+{
+	@NotNull
+	@Getter
+	@Setter
```

Comment:
Why are you using those annotations here ?

Reply:
It was a mistake, those @Getter and @Setter should not be there, I'm going to remove them